resources/assets/uad_lists.json

```
@@ -10495,12 +10495,12 @@
   },
   {
     "id": "com.samsung.android.mcfds",
-    "list": "Pending",
-    "description": "",
+    "list": "OEM",
```

```suggestion
    "list": "Oem",
```
Pretty sure it's case sensitive, so it should be like this instead.